get_element_cpo/P2769.bs

```diff
@@ -606,8 +613,7 @@ semantics, for example, `std::less` and `std::ranges::less`, where the latter is
 
 # Further work # {#further_work}
 
-- <s>Substitute std::get_element for std::get in formal wording of the APIs with tuple-like</s>
-- Broader implementation experience
+- Broader implementation experience (should be done before meeting in Poland, 2024)
```

Comment:
Does it make sense to add a question of making the tuple-like concept public here? Not as further work but as a question to discuss.
Also, in addition to / instead of the concept. would a type trait `is_tuple_like` make sense? Should that be also discussed?

Reply:
I think we should at least discuss making it not exposition-only. and personally I think it should *not* be exposition only.

there is no need for both a concept and a type trait. the concept is enough